leshan-client-cf/src/main/java/org/eclipse/leshan/client/californium/LeshanClient.java

```diff
@@ -230,6 +247,13 @@ public void destroy(boolean deregister) {
         engine.destroy(deregister);
         endpointsManager.destroy();
         requestSender.destroy();
+
+        for (LwM2mObjectEnabler objectEnabler : objectTree.getObjectEnablers().values()) {
+            if (objectEnabler instanceof Destroyable) {
+                ((Destroyable) objectEnabler).destroy();
+            }
+        }
```

Comment:
For destroy there is a subtility in case use implement `Stoppable` but not `Destroyable`. See [LeshanServer](https://github.com/eclipse/leshan/blob/leshan-1.2.0/leshan-server-cf/src/main/java/org/eclipse/leshan/server/californium/LeshanServer.java#L380)

Reply:
Gotcha, I'm going to fix this.

Reply:
fixed at bd7f86d